dashboards_test.go

```diff
@@ -23,11 +23,29 @@ func TestHostIFrameGraph(t *testing.T) {
 }
 
 func TestServiceIFrameGraph(t *testing.T) {
```

Comment:
[nit] In the other file `Iframe` seems to be used in function names, but here `IFrame` is used.